swap_meet/item.py

```diff
@@ -1,2 +1,69 @@
 class Item:
-    pass
\ No newline at end of file
+    '''
+    A class to represent an item object
+
+    ...
+
+    ---Attributes---
+    category : str, optional
+        empty string by default, otherwise a specific category represented by a string
+    condition : int or float, optional
+        zero by default, otherwise a number representing the condition of the item
+    age : int or float, optional
+        None by default, otherwise a number representing how old the item is in years
+
+    ---Methods---
+    __str__():
+        sets dunder str to a string
+
+    condition_description():
+        based on an item instance's numerical condition, returns a string description of that condition
+
+    '''
+    def __init__(self, category="", condition=0, age=None):
+        '''
+        Constructs all the necessary attributes for the item object
+
+        ---Paramters---
+        category : str, optional
+            empty string by default, otherwise a specific category represented by a string
+        condition : int or float, optional
+            zero by default, otherwise a number representing the condition of the item
+        age : int or float, optional
+            None by default, otherwise a number representing how old the item is in years
+        '''
+        self.category = category
+        self.condition = condition
+        self.age = age
+
+
+    def __str__(self):
+        '''sets dunder str to a string'''
+        return "Hello World!"
+
+    def condition_description(self):
+        '''
+        based on an item instance's numerical condition, returns a string description of that condition
+        '''
+        descriptors = [
+            "Mint",
+            "Like New",
+            "Gently Used",
+            "Used",
+            "Well Used",
+            "Bad Condition",
+        ]
+        if 1 > self.condition >= 0:
```

Comment:
Love that you put the descriptions in a list (data-driven). Once you do that, try to write an approach that doesn't require essentially using each position as an individual hard-coded value. For instance, if we added an additional condition, we would have to add additional new clauses in the if chain to check for the new condition. What if instead we could either loop through the list, or somehow otherwise convert the condition into an index we could use?